OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -368,7 +362,7 @@ public Sprite GetShadow(int frame, int facing)
 
 		protected virtual Sprite GetSprite(int start, int frame, int facing)
 		{
-			var f = Util.QuantizeFacing(facing, Facings, useClassicFacingFudge);
+			var f = Util.QuantizeFacing(facing, Facings);
```

Comment:
It would be cleaner if we could define a
```csharp
protected virtual int QuantizeFacing(int facing)
{
	return Util.QuantizeFacing(facing, Facings);
}
```
to call here and override in the subclass. This would save us from duplicating the rest of the calculations below.